src/sqlittaaja/diff_check.py

```diff
@@ -0,0 +1,47 @@
+import difflib
+from typing import Any
+from zipfile import ZipFile
+from sqlittaaja.extractor import extract, student_info
+
+# Threshold percentage for warning >=
+threshold = 90
+
+
+def compare_files(file1_content, file2_content):
+    """Compute the similarity ratio using SequenceMatcher"""
+
+    similarity_ratio = (
+        difflib.SequenceMatcher(None, file1_content, file2_content).ratio() * 100
+    )
+    return similarity_ratio
+
+
+def compute_similarity(config: dict[str, Any]) -> dict[str, int]:
+    """Check students' exercises."""
+
+    similarity_matrix = {}
+
+    with ZipFile(config["exercise"]["path"], "r") as exercises:
```

Comment:
We have already read and parsed the ZIP-file so should we maybe use the existing data here?

Reply:
I had some original thought, that maybe the check could be run independently with some argument from cmd-line if needed, but propably not relevant atm.